openwisp_radius/tests/test_api.py

```diff
@@ -989,7 +989,7 @@ def test_batch_csv_pdf_link_404(self):
             response = self._radius_batch_post_request(data)
         self.assertEqual(response.status_code, 201)
         response_json = json.loads(response.content)
-        org = Organization.objects.get(pk=response_json['organization'])
+        org = Organization.objects.get(slug=response_json['organization'])
```

Comment:
The API returns organzation slug instead of pk because I think that should be more user friendly, what do you think?